packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -388,6 +389,43 @@ class RelativePositionedTransition extends AnimatedWidget {
   }
 }
 
+/// Animated version of a [DecoratedBox] that animates the different properties
+/// of its [Decoration].
+class DecoratedBoxTransition extends AnimatedWidget {
+  /// Creates an animated [DecorationBox] whose [Decoration] animation updates
+  /// the widget. 
+  /// 
+  /// `decoration` and `position` cannot be null.
```

Comment:
Ah it links to the property. Was wondering why there was half and half in the other files.